<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voronoi diagram</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>615</width>
      <height>615</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Run">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>710</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Step">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>710</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Step by step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Load">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>710</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Set">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_X">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Y">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_X">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Y">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
